py/legacyanalysis/sersic-agn.py

```diff
@@ -0,0 +1,269 @@
+        # stack.append(wzero)
+        stack.append(np.hstack(panels[:cols]))
+        panels = panels[cols:]
+    stack = np.vstack((stack))
+    ax.imshow(stack, vmin=-2, vmax=2, **ima)
+    xl,yl = ax.get_xlim(), ax.get_ylim()
+    #a = ax.get_axis()
+    for c in range(cols+1):
+        plt.axvline(c * w, color='k')
+    for r in range(rows+1):
+        plt.axhline(r * h, color='k')
+    #ax.set_axis(a)
+    ax.set_xlim(xl)
+    ax.set_ylim(yl)
+    #xl,yl = ax.get_xlim(), ax.get_ylim()
+    ax.set_xticks(w * (0.5 + np.arange(cols)))
+    ax.set_xticklabels(np.arange(cols))
+    ax.set_yticks(h * (0.5 + np.arange(3)))
+    ax.set_yticklabels(['g','r','z'])
+
+    ax = fig.add_subplot(gs[2, 3])
+    co,comod = showresid(tims, mods, wcs)
+    ax = fig.add_subplot(gs[0, 3])
+    plt.imshow(np.flipud(get_rgb(co, bands)))
+    plt.xticks([]); plt.yticks([])
+    ax = fig.add_subplot(gs[1, 3])
+    plt.imshow(np.flipud(get_rgb(comod, bands)))
+    plt.xticks([]); plt.yticks([])
+
+def main():
+
+    if os.path.exists('results.pickle'):
+        results = unpickle_from_file('results.pickle')
+    else:
+        results = []
+
+
+    
+    for isrc,(ra,dec,brickname) in enumerate([
+        (0.2266, 3.9822, '0001p040'),
+        (7.8324, 1.2544, '0078p012'),
+        (1.1020, 3.9040, '0011p040'),
+        (7.3252, 4.6847, '0073p047'),
+        (3.1874, 3.9724, '0031p040'),
+        (9.5112, 4.6934, '0094p047'),
+        (4.4941, 1.1058, '0043p010'),
+        (3.8900, 0.6041, '0038p005'),
+        (8.1934, 4.0124, '0081p040'),
+        (6.8125, 0.5463, '0068p005'),
+        ]):
+
+        #if isrc < 7:
+        #    continue
+        if isrc not in [4,6,7,8,9]:
+            continue
+        
+        outdir = 'out_%.4f_%.4f' % (ra,dec)
+        datadir = outdir.replace('out_', 'data_')
+        #cmd = 'ssh cori "cd legacypipe2/py && python legacypipe/runbrick.py --radec %.4f %.4f --width 100 --height 100 --survey-dir fakedr9 --outdir %s --stage image_coadds --skip-calibs && python legacyanalysis/create_testcase.py --survey-dir fakedr9 %s/coadd/*/*/*-ccds.fits %s %s"' % (ra, dec, outdir, outdir, datadir, brickname)
+        #cmd = 'ssh cori "cd legacypipe2/py && python legacypipe/runbrick.py --radec %.4f %.4f --width 100 --height 100 --survey-dir fakedr9 --outdir %s --stage image_coadds --skip-calibs && python legacyanalysis/create_testcase.py --survey-dir fakedr9 --outlier-dir %s %s/coadd/*/*/*-ccds.fits %s %s"' % (ra, dec, outdir, outdir, outdir, datadir, brickname)
+
+        outbrick = ('custom-%06i%s%05i' %
+                    (int(1000*ra), 'm' if dec < 0 else 'p',
+                     int(1000*np.abs(dec))))
+
+        cmd = 'ssh cori "cd legacypipe2/py && python legacyanalysis/create_testcase.py --survey-dir fakedr9 --outlier-dir %s --outlier-brick %s %s/coadd/*/*/*-ccds.fits %s %s"' % (outdir, outbrick, outdir, datadir, brickname)
+        #os.system(cmd)
+    
+        cmd = 'rsync -arv cori:legacypipe2/py/%s .' % datadir
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'cmd' is assigned to but never used](https://app.codacy.com/app/dstndstn/legacypipe/pullRequest?prid=4730106)